### PR TITLE
[FIX] Add VAT on partner tree view & fix tax group widget on account move

### PR DESCRIPTION
**Pull request by FlorianGilbert**
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by FlorianGilbert** [comment]
@qdp-odoo @csnauwaert @oco-odoo @smetl One of you could review this pull request please ?

File: addons/account/models/account_move.py [hunk 1]
```diff
@@ -1239,7 +1239,7 @@ def _compute_invoice_taxes_by_group(self):
             done_taxes = set()
             for line in tax_lines:
                 res.setdefault(line.tax_line_id.tax_group_id, {'base': 0.0, 'amount': 0.0})
-                res[line.tax_line_id.tax_group_id]['amount'] += line.price_subtotal
+                res[line.tax_line_id.tax_group_id]['amount'] += line.currency_id and line.amount_currency or line.price_subtotal
```
Comment (smetl) [inline addons/account/models/account_move.py]: There is something wrong here:
balance = debit - credit (company's currency)
amount_currency = balance in foreign currency
price_subtotal = signed(amount_currency or balance) depending the invoice's currency and the invoice's type.

In your case, `line.price_subtotal` is correct.

File: addons/account/static/src/js/tax_group.js [hunk 2]
```diff
@@ -39,19 +39,28 @@ odoo.define('account.tax_group', function (require) {
...
                 creditAmount = -line_id.data.credit - deltaAmount; 
             }
-            // If debitAmount is negative, the value is added to the credit.
-            if (debitAmount < 0) {
-                creditAmount = Math.abs(debitAmount);
-                debitAmount = 0;
-            } else if (creditAmount > 0) {
-                debitAmount = Math.abs(creditAmount);
-                creditAmount = 0;
+            var amount_currency = 0;
```
Comment (smetl) [inline addons/account/static/src/js/tax_group.js]: These conditions should be dependent of the invoice's type but... you don't have this value I guess...

File: odoo/addons/base/views/res_partner_views.xml [hunk 3]
```diff
@@ -52,6 +52,7 @@
                     <field name="city" optional="hide"/>
                     <field name="state_id" optional="hide"/>
                     <field name="country_id" optional="hide"/>
+                    <field name="vat" optional="hide"/>
```
Comment (smetl) [inline odoo/addons/base/views/res_partner_views.xml]: gné?

Comment (FlorianGilbert) [reply odoo/addons/base/views/res_partner_views.xml]: because AL

**Comment by smetl** [comment]
@robodoo r+

**Pull request closed without merging**